app/models/drawing.rb

```diff
@@ -49,6 +52,14 @@ def self.to_csv(hxl: false)
     end
   end
 
+  def self.stages_of_journey
+    @stages_of_journey = ["At home", "In temporary shelter", "Awaiting transit", "On the move", "Arrived at destination"]
+  end
+
+  def self.subject_matters
+    @subject_matters = ["Home / Country of origin", "In transit", "Camp life", "Future_hopes / destination"]
```

Comment:
After discussing with Kris last night on the most Ruby/Rails way of doing this, can you change these to reference just the constant STAGES_OF_JOURNEY, and move the array back up to line 3?

Line 3:

STAGES_OF_JOURNEY = ["Home / Country of origin", "In transit", "Camp life", "Future_hopes / destination"].freeze
Line 59-61:

def self.subject_matters
  SUBJECT_MATTERS
end